CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -370,8 +376,32 @@ def get_item(self, fridge_id):
                 json_data=json_data,
             )
 
-    def get_items(self):
-        pass
+    def get_items(self, tag=None):
+        # scan/query doc: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/dynamodb.html#querying-and-scanning
+        response = None
+        if tag:
+            response = self.db_client.scan(
+                TableName=self.TABLE_NAME,
+                FilterExpression="contains (tags, :tag)",
+                ExpressionAttributeValues={":tag": {"S": tag}},
+                ProjectionExpression="json_data",
+            )
+        else:
+            response = self.db_client.scan(
+                TableName=self.TABLE_NAME, ProjectionExpression="json_data"
+            )
+        items = response["Items"]
```

Comment:
nit: List comprehension?
```python
data = [item["json_data"]["S"] for item in response["Items"]]
```

Reply:
yeah! Lets use it :)

Reply:
done: 74837d711b0ddcc1529ee93827aaf2a7b073e15b